src/System.Management.Automation/engine/parser/PSType.cs

```diff
@@ -17,6 +17,7 @@ namespace System.Management.Automation.Language
     internal class TypeDefiner
     {
         internal const string DynamicClassAssemblyName = "PowerShell Class Assembly";
+        internal const string DynamicClassAssemblyFullName = "PowerShell Class Assembly,";
```

Comment:
This is not a full name, so the variable name is not appropriate.
And I don't think this string will be used anywhere else, so why not move it to `GetAssemblies(string namespaceQualifiedTypeName = null)` just like what I showed in my example?

Reply:
I think both const must be near - so it is more clear to code readers that they belong to the same entity.

The const is renamed to DynamicClassAssemblyFullNamePrefix.